src/main/java/nextstep/sessions/service/SessionService.java

```diff
@@ -24,4 +24,18 @@ public void enroll(Payment payment) {
         SessionStudent sessionStudent = session.enroll(optionalUser.get());
         sessionRepository.enroll(session, sessionStudent);
     }
+
+    @Transactional
+    public void approval(Session session, NsUser enrollUser, NsUser loginUser) {
+        SessionStudent student = sessionRepository.studentFindBySessionIdAndUserId(session.getId(), enrollUser.getId());
+        session.approval(student, loginUser);
+        sessionRepository.approvalStudent(session, student);
```

Comment:
```suggestion
        sessionRepository.approvalStudent(student);
```
굳이 session을 인자로 넘길 필요가 있을까?